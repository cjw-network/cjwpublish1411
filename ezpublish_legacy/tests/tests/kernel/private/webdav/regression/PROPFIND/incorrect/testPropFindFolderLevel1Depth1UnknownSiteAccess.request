<?php

// Test if PROPFIND returns '404 Not Found' for the Content folder
// with an unknown siteaccess

$_SERVER['REQUEST_URI'] = '/this_siteaccess_cannot_possibly_exist/Content/';
$_SERVER['REQUEST_METHOD'] = 'PROPFIND';

$_SERVER['HTTP_DEPTH'] = '1';
$_SERVER['CONTENT_TYPE'] = 'application/xml';
$_SERVER['HTTP_CONTENT_LENGTH'] = '288';

// The PROPFIND request needs an XML body
$GLOBALS['ezc_post_body'] = <<<END
<?xml version="1.0" encoding="utf-8"?>
<propfind xmlns="DAV:"><prop>
<creationdate xmlns="DAV:"/>
<getcontentlength xmlns="DAV:"/>
<getcontenttype xmlns="DAV:"/>
<getlastmodified xmlns="DAV:"/>
<displayname xmlns="DAV:"/>
<executable xmlns="http://apache.org/dav/props/"/>
<resourcetype xmlns="DAV:"/>
<supportedlock xmlns="DAV:"/>
<checked-in xmlns="DAV:"/>
<checked-out xmlns="DAV:"/>
</prop></propfind>
END;

?>
