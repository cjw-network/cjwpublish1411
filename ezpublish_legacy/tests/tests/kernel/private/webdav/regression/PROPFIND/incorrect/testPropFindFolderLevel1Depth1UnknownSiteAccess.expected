Server: eZComponents/dev/ezcWebdavTransportWrapper
Content-Type: text/plain; charset="utf-8"
Content-Length: 312
HTTP/1.1 404 Not Found

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
                <html><head>
                <title>404 Not Found</title>
                </head><body>
                <h1>Not Found</h1>
                <p>The requested URL was not found on this server.</p>
                <hr>
                </body></html>
